vipps-report-api-faq.md

```diff
@@ -58,6 +60,28 @@ or on
 The only difference is that the data can be fetched over a more modern REST API.
 We aim to provide more information through this API in the future.
 
+## What are the benefits of the Report API over the SFTP service?
+
+The
+[Vipps SFTP Report Service](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/settlements/sftp-report-service)
+lets merchants and partners download settlement files by SFTP.
+
+The Report API has some benefits over this:
+- Data can be retrieved at any time.
+- Data is available regardless of whether the balance is positive or negative,
```

Comment:
Actually this is more of a planned feature, but there is a "bug" now that means that this isn't solved.

Reply:
We'll comment it out for now and then it won't show up. We can update it more when the bug is fixed. :-)